pkg/commands/import/import.go

```diff
@@ -98,8 +101,14 @@ cat dependencies.yaml | kp import -f -`,
 			}
 
 			defaultKeychain := authn.DefaultKeychain
+			kc := authn.NewMultiKeychain(
+				defaultKeychain,
```

Comment:
We may want to flip the order of these 2 keychains so the default keychain is the fall back

Reply:
Addressed in commit https://github.com/vmware-tanzu/kpack-cli/pull/279/commits/4f878aa8074dd4243e69bf233be09de7b66c2cb9
And removed commented out code in https://github.com/vmware-tanzu/kpack-cli/pull/279/commits/eee5b28ef0778d4595e9ad03a0da08145d21accb